test/t8n/t8n.cpp

```diff
@@ -160,6 +160,10 @@ int main(int argc, const char* argv[])
             if (block_reward.has_value())
                 state.touch(block.coinbase).balance += *block_reward;
 
+            // Apply withdrawals. Amount value is in gwei.
+            for (const auto& withdraw : block.withdrawals)
+                state.touch(withdraw.first).balance += withdraw.second * 1000000000;
```

Comment:
Are withdrawals applies before block finalization? I.e. are touched accounts possibly removed?

Reply:
Spec says that `amount` is non-zero. So I will not be removed on finalizing. Am I right? https://eips.ethereum.org/EIPS/eip-4895